frontend/src/pages/EventSignInPage/components/EventSignInForm/index.js

```diff
@@ -22,19 +26,58 @@ const INITIAL_INPUT_VALUES = {
 const EventSignInForm = props => {
   const { classes, handleSubmit } = props;
 
+  const [modalDisplay, setModalDisplay] = useState(false);
+
+  const history = useHistory();
```

Comment:
No need for spaces between short const initialization like this. Group the const statements on lines 27, 29 and 31 together and put them at the beginning of the function body, with the hooks going before the destructuring statement (useState first, then useHistory)